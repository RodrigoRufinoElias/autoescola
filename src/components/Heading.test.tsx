import React from "react";
import { render } from "@testing-library/react";
import Heading from "./Headings";

describe("Heading Component", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<Heading>Title</Heading>);
    expect(asFragment()).toMatchSnapshot();
  });
});
