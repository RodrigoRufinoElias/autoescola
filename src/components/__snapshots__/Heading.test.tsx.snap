// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading Component should match snapshot 1`] = `
<DocumentFragment>
  .c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6 {
  position: relative;
  margin-bottom: 25px;
  padding-bottom: 8px;
  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
}

.c0 h1::after,
.c0 h2::after,
.c0 h3::after,
.c0 h4::after,
.c0 h5::after,
.c0 h6::after {
  content: "";
  position: absolute;
  width: 70px;
  height: 5px;
  background-color: #ffc107;
  bottom: -3px;
  left: 0;
}

.c0 h1 {
  padding-bottom: 25px;
}

<div
    class="c0"
  >
    Title
  </div>
</DocumentFragment>
`;
